app/models/SprintSnapshot.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class SprintSnapshot extends Eloquent {
+	protected $fillable =  ['data', 'sprint_id'];
+
+	public function sprint()
```

Comment:
`getSprint` or something like `getParentSprint`? Also, what does this return? A `Sprint` or some integer id?